app/src/app/worker/hybrid.rs

```diff
@@ -53,14 +54,14 @@ impl WorkerApp for HybridWorkerAppImpl {
             tx.commit().await.map_err(JobWorkerError::DBError)?;
             id
         };
-        let w = Worker {
-            id: Some(wid.clone()),
-            data: Some(worker.clone()),
-        };
-        self.redis_worker_repository().upsert(&w, false).await?;
+        let _ = self.redis_worker_repository().delete_all().await;
```

Comment:
when expire redis cache, redis cache broken with upsert.
instead, only delete all for cache